### DISCUSSION TITLE
GitHub Student Developer Pack - outstanding issues with my GitHub

### DISCUSSION THREAD

**Discussion by poil12** [community/community#43804] [GitHub Education]
[GitHub Student Developer Pack - github copilot](https://github.com/orgs/community/discussions/32199)
This problem arises how to solve it，As shown in the image，There are outstanding issues with your GitHub account. Please contact GitHub Support for more information. As shown in the image
I have completed a verification，but because of the location issue I was asked to reverify.
But it says
"There are outstanding issues with your GitHub account. Please contact GitHub Support for more information."
What should I do

**Comment by jongwooo** [comment]
@poil12 You can create related ticket on [GitHub Support](https://support.github.com/request) portal.

**Reply by poil12** [reply]
actually，
![image](https://user-images.githubusercontent.com/112222244/211507237-82f2766e-3e6a-4e14-b0f0-f08578cc887d.png)
They were the ones who asked me to come here to solve the problem

**Comment by designlandwebsolutions** [comment]
Same to me :(
and they asked me to come here too

**Comment by HorselessName** [comment]
Hey I went there and was asked to come here
I would like to use GitHub Copilot but am not being able to renew my student pack.

My school does not give me emails and I live in Brazil on a lower class, so can not afford anything and this pack really helps me.
But no one is answering me on tickets :(

**Comment by Sameer9619** [comment]
there are outstanding issues with your github account. please contact github support for more information.how can I fix it please
